libs/wingc/src/visit.rs

```diff
@@ -254,22 +257,29 @@ where
 	}
 }
 
+pub fn visit_expr_new<'ast, V>(v: &mut V, _node: &'ast Expr, class: &'ast TypeAnnotation, _obj_id: &'ast Option<String>, obj_scope: &'ast Option<Box<Expr>>, arg_list: &'ast ArgList)
+	where
+	V: Visit<'ast> + ?Sized,
+{
+	v.visit_type_annotation(class);
+	if let Some(scope) = obj_scope {
+		v.visit_expr(&scope);
+	}
+	v.visit_args(arg_list);
```

Comment:
Feels like the args should come before the `scope`.
```suggestion
	v.visit_args(arg_list);
	if let Some(scope) = obj_scope {
		v.visit_expr(&scope);
	}
```